.github/workflows/ci.yml

```diff
@@ -144,4 +144,3 @@ jobs:
         - name: Run pipeline with smoothxg and various parameters
           run: |
             nextflow run ${GITHUB_WORKSPACE} -profile test,docker ${{ matrix.parameters }}                        
-            
```

Comment:
Now there really is no EOL :)
I verified it.

Reply:
![image](https://user-images.githubusercontent.com/10184961/100904417-4bd02300-34c7-11eb-91b2-70f05372d701.png)
My editor says otherwise, hmmmmmm

Reply:
This is resolved now. I don't know why Visual Studio is lying to me.